src/features/collapsible-sidebar/CollapsibleSidebar.js

```diff
@@ -36,22 +37,66 @@ type Props = {
     htmlAttributes?: Object,
 };
 
-const CollapsibleSidebar = (props: Props = { expanded: false }) => {
-    const { children, className, expanded, htmlAttributes } = props;
-
-    const classes = classNames(
-        {
-            'is-expanded': expanded,
-        },
-        'bdl-CollapsibleSidebar',
-        className,
-    );
-
-    return (
-        <aside className="bdl-CollapsibleSidebar-wrapper" {...htmlAttributes}>
-            <StyledNav className={classes}>{children}</StyledNav>
-        </aside>
-    );
-};
+class CollapsibleSidebar extends React.Component<Props> {
+    navRef: { current: null | HTMLElement } = React.createRef();
+
+    focusNextEl = () => {
+        if (this.navRef.current) {
+            const tabbableEls = tabbable(this.navRef.current);
+            const currentElIndex = tabbableEls.findIndex(el => el === document.activeElement);
+            const nextElIndex = currentElIndex === tabbableEls.length - 1 ? 0 : currentElIndex + 1;
+            tabbableEls[nextElIndex].focus();
+        }
+    };
+
+    focusPreviousEl = () => {
+        if (this.navRef.current) {
+            const tabbableEls = tabbable(this.navRef.current);
+            const currentElIndex = tabbableEls.findIndex(el => el === document.activeElement);
+            const prevElIndex = currentElIndex === 0 ? tabbableEls.length - 1 : currentElIndex - 1;
+            tabbableEls[prevElIndex].focus();
+        }
+    };
+
+    handleKeyDown = (event: SyntheticKeyboardEvent<>) => {
+        if (this.navRef.current && this.navRef.current.contains(document.activeElement)) {
+            switch (event.key) {
+                case 'ArrowDown':
```

Comment:
Should probably use `KEYS.arrowDown` from `src/constants` and similar for `arrowUp`.